sklearn/linear_model/stochastic_gradient.py

```diff
@@ -1341,3 +1354,416 @@ def __init__(self, loss="squared_loss", penalty="l2", alpha=0.0001,
                                            eta0=eta0, power_t=power_t,
                                            warm_start=warm_start,
                                            average=average, n_iter=n_iter)
+
+
+class SGDOneClassSVM(BaseSGD):
+    """Solves linear One-Class SVM using Stochastic Gradient Descent.
+
+    Parameters
+    ----------
+    nu : float, optional
+        The nu parameter of the One Class SVM: an upper bound on the
+        fraction of training errors and a lower bound of the fraction of
+        support vectors. Should be in the interval (0, 1]. By default 0.5
+        will be taken.
+
+    fit_intercept : bool
+        Whether the intercept should be estimated or not. Defaults to True.
+
+    max_iter : int, optional
+        The maximum number of passes over the training data (aka epochs).
+        It only impacts the behavior in the ``fit`` method, and not the
+        `partial_fit`.
+        Defaults to 5. Defaults to 1000 from 0.21, or if tol is not None.
```

Comment:
I guess as this class is new, we can directly use the future default value.
what do you think?

Reply:
done